### DISCUSSION TITLE
Custom types in remote schema.

### DISCUSSION THREAD

**Discussion by 3goats** [hasura/graphql-engine#8914] [General] [answered]
Not quite sure how to explain this question, but here goes. 

I have remote schema that get introspected like this. 

![image](https://user-images.githubusercontent.com/555399/188142816-f7c45eac-eba9-4b5f-93b4-1771bf82bca5.png)

The `fqdn` field has a type of `venaf_query_DisplayableString` which consists of a `value` and `display_value`. 

The `fqdn` field is defined in a relationship, however when I run my query it fails with this error: 

```
{
  "errors": [
    {
      "extensions": {
        "code": "unexpected",
        "internal": {
          "errors": [
            {
              "extensions": {
                "code": "GRAPHQL_VALIDATION_FAILED"
              },
              "locations": [
                {
                  "column": 883,
                  "line": 1
                }
              ],
              "message": "Field \"value\" is not defined by type \"String\"."
            }
          ]
        },
        "path": "$"
      },
      "message": "Errors from remote server"
    }
  ]
}
```

So from what I can see Hasura is expecting a String but not getting that because the schema is a diffrent type. 

Any ideas?

**Comment by adas98012** [comment, marked as answer]
The issue you are seeing looks related to one logged recently (concerned with Remote Schema):
https://github.com/hasura/graphql-engine/issues/8898

Please follow the above issue for any updates related to a fix. You can also look at the workaround mentioned in the issue, until a fix is available. Thank you for your patience.
